src/store/trees/trees.js

```diff
@@ -162,19 +160,24 @@ export default {
 
   mutations: {
 
+    // TODO: find a better place to put the configure, putting it inside trees store doesn't make sense
     configure({ui}, {appConfig, builderConfig}) {
       const {
         app,
         __AUDIO__: audio,
         __TREES_UI__: uiOverrides,
+        __APP__: appContext,
       } = bootstrapLegacyGlobalDependencies(appConfig, builderConfig)
 
+      // update this.state to expose permissions, etc
+      lodash.merge(this.state, appContext)
+      lodash.merge(this.state.audio, audio)
+
       // todo: audio recording feature is likely to be unavailable for standalone app - How do we want to isolate these?
       set(app, 'audioChoice.audioLibrary', audio.library)
```

Comment:
@jcbashdown , can you remind me why we set these `audioChoice` in the `app` ? Are states from `audio.js` not enough ? I'm wondering if we should keep these 02 lines

Reply:
I'm not sure. Looks like I copied this wholesale from the original function without really understanding the logic. 

I'm a bit confused though, could you point me to where in audio.js we also set these up? Can't find the exact line.

Reply:
Hi hi, we don't need to do this monkey business anymore. This was backwards compat for tree builder. We can leave it for now and tackle it as part of https://viamoinc.atlassian.net/browse/VMO-3577